### ISSUE TITLE
License issue - CockroachDB

### ISSUE THREAD

**Issue by janotav** [jacksontj/promxy#207]
Hi, we would like to use promxy but we seem to be affected by the following (upstream) licensing issue: https://github.com/prometheus/prometheus/issues/5783

Could you please release version of promxy that does not depend on CockroachDB (through Prometheus)?

**Comment by jacksontj** [comment]
From a quick look it seems that prom 2.12 (supposed to be released today) will remove that dep; if so it should be pretty straight-forward to do. Once that release is done I'll take a look at updating the prom dep.

**Comment by janotav** [comment]
Great. For us this is a blocker issue and we currently parked all our efforts around promxy until this is resolved.

**Comment by jacksontj** [comment]
I misread the prom docs, it seems that today is rc0 day; I took a look and the backport was trivial, so I just made the change. #208 . I've also cut a [release](https://github.com/jacksontj/promxy/releases/tag/v0.0.47) with the dep removed

**Comment by janotav** [comment]
Nice. I'll get back on to this on Monday. Thanks!
